src/help-widget.ts

```diff
@@ -39,11 +42,13 @@ export class BBHelpHelpWidget {
   }
 
   public load(config: HelpConfig) {
-
     if (this.loadCalled) {
       return;
     }
 
+    this.analyticsService.config = config;
+    this.analyticsService.setupMixpanel();
```

Comment:
Since the analytics service only needs the `config.productId` and only once... can't we just make that a parameter value on `setupMixpanel`?   `this.analyticsSerice.setupMixpanel(config.productId)`?

Then we don't need that service to store another reference to a full config it doesn't care about.